Automattic/wp-calypso
# Login: migrate CSS to webpack

A rather large-scale cleanup of login styles and markup. Migrates CSS, moves a lot of styles to separate per-component stylesheets, converts a few components to hookified functional ones, does several a11y fixes, fixes broken styles on Woo and CrowdSignal login pages.

I tried to do small commits where possible, so refer to commit messages if you choose to review the diff thoroughly.

**How to test:**
There are several variants of the login flow:
- classic WordPress.com
- WooCommerce.com login
- CrowdSignal.com login
- login during Jetpack Connect
- login during Jetpack + Woo Onboarding (see also #32993)

Check that login including the 2FA steps is correctly styled for all of them.

Things to check in the login flow:
- entering username and password
- the social login box ("Continue with Google")
- "waiting for push notification" screen (awesome design by @porada BTW!)
- buttons to choose to do 2FA with another method (OTP code, SMS message)
